proto/multistaking/v1/query.proto

```diff
@@ -34,6 +34,9 @@ service Query {
     option (google.api.http).get =
         "/realiotech/multistaking/v1/validator-coin/{validator_addr}";
   }
+  rpc BondWeights(QueryBondWeightsRequest) returns (QueryBondWeightsResponse) {
```

Comment:
I think this should be some thing like `MultiStakingCoinsInfo` because we not just focus on the `weight` of `bond token`. Instead, we also want to know `denom` of `bond token`

Reply:
could you review my PR again @GNaD13?